atom/browser/node_debugger.cc

```diff
@@ -37,15 +37,14 @@ void NodeDebugger::Start() {
     // Use custom platform since the gin platform does not work correctly
     // with node's inspector agent
     platform_.reset(new node::NodePlatform(
-        /* thread_pool_size */ 0,
-        env_->event_loop(),
+        /* v8_default_thread_pool_size */ 4, env_->event_loop(),
```

Comment:
Actually I would _love_ to keep every argument on its own line, it would make diffs smaller and
also would preserve correct blame. Not a big deal though =)

Reply:
Yeah it was `clang-format` :( Should I revert it ?

Reply:
I guess there is no need to revert it. Having a properly formatted code is a good thing too.